### ISSUE TITLE
Can't switch to the right panel when installing

### ISSUE THREAD

**Issue by MichaelMa2014** [VundleVim/Vundle.vim#797]
Hitting <Ctrl>+w+l doesn't work.

**Comment by MichaelMa2014** [comment]
Hitting `<Ctrl>+w+l` doesn't work.

**Comment by davidlowryduda** [comment]
This is true. The installation doesn't occur asynchronously (although the plugin neobundle was based on Vundle, and includes async installation; and dein, which is a spiritual successor to neobundle, goes one step further).

**Comment by specious** [comment]
I've used vundle for ages. Just switched to [dein](https://github.com/Shougo/dein.vim).
